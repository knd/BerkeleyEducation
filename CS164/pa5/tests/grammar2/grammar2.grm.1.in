move ANH to LEFT 5