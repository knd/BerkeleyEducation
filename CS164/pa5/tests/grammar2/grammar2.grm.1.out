('move', 'ANH', ('LEFT', 5))
